types/configs.d.ts

```diff
@@ -0,0 +1,11 @@
+import type { Linter } from "eslint"
+
+declare namespace Configs {
+    import defaultExports = Configs
+
+    export const recommended: Linter.Config
```

Comment:
fwiw I'm pretty sure it would be better to use `Linter.FlatConfig` for now, as in `@types/eslint` v8 the `Linter.Config` type is the legacy config format whereas in the v9+ types it's the flat config type

Note that while `Linter.FlatConfig` is marked as deprecated, that shouldn't be a concern here as it's an alias to help folks transition from legacy to flat configs (likewise the v8 types have the deprecated `Linter.LegacyConfig` type which is an alias as well).

I think technically this would mean the types cannot be used with versions lower than v8 as I don't think the aliases got backported, but I don't think that'll actually be an issue given there won't be any reason to import plugins without flat config.

Reply:
Done